core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -258,11 +258,11 @@ const getSkillEditorUrl = async function(browser, page) {
     await page.type(skillDescriptionField, 'Skill Description here');
     await page.click(skillOpenConceptCard);
     await page.waitForSelector(skillReviewMaterialInput, {visible: true});
-    await page.waitForTimeout(5000);
+    await page.waitForTimeout(10000);
     await page.keyboard.type('Skill Overview here');
```

Comment:
@jimbyo -- it seems like we have narrowed down to `page.keyboard.type('Skill Overview here');` not happening as expected and the test clicking on `confirmSkillCreationButton` when it isn't ready. Is this correct?

If so, do we know _why_ is it that `page.keyboard.type(..)`  is failing? e.g. is it that the RTE toolbar is missing or that the field is not focused on time or something else?

Reply:
Yes, sometimes it takes awhile to switch focus to the

Reply:
Hm, if the problem is that the element is not focused in time. Can we use puppeteer's waitForFunction() ([here](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagewaitforfunctionpagefunction-options-args)) to wait for the element to be focused before typing?